ci/azure/pipelines.yml

```diff
@@ -91,15 +91,22 @@ jobs:
       $Env:AWS_EC2_METADATA_DISABLED = "true"
       $Env:AWS_REGION = "us-west-2"
 
-      mv LICENSE stage3-release/
-      mv zig-cache/langref.html stage3-release/
-      mv stage3-release/bin/zig.exe stage3-release/
-      rmdir stage3-release/bin
-
-      # Remove the unnecessary zig dir in $prefix/lib/zig/std/std.zig
-      mv stage3-release/lib/zig stage3-release/lib2
-      rmdir stage3-release/lib
-      mv stage3-release/lib2 stage3-release/lib
+      # Produce the experimental std lib documentation.
+      mkdir "$ZIGINSTALLDIR\doc\std" -force
+      & "$ZIGINSTALLDIR\bin\zig.exe" test "$ZIGLIBDIR\std\std.zig" `
+        --zig-lib-dir "$ZIGLIBDIR" `
+        -femit-docs="$ZIGINSTALLDIR\doc\std" `
+        -fno-emit-bin
```

Comment:
Maybe this would be better in the 'Test' step which builds the docs after `& "$ZIGINSTALLDIR\bin\zig.exe" build test docs` instead of the 'Upload' step.